templates/taxi/car_list.html

```diff
@@ -7,6 +8,14 @@ <h1>
       Create
     </a>
   </h1>
+
+  <form action="" method="get" class="form-inline">
+      {{ search_form|crispy }}
+      <input type="submit" value="🔎" class="btn btn-secondary">
+  </form>
```

Comment:
Create a separate template for the search form, and use this template where you need a search form